2.4/test/run-conu

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exec pytest-3 -vv ./test_s2i.py
```

Comment:
This needs to be: `exec pytest-3 -vv ./test/test_s2i.py`
The test is being run from `$ROOT/$VERSION`